targets/LumberyardSdk/make.js

```diff
@@ -751,7 +751,7 @@ function GetAuthParams(apiCall) {
     if (apiCall.auth === "SecretKey")
         return "\"X-SecretKey\", PlayFabSettings::playFabSettings->developerSecretKey";
     else if (apiCall.auth === "SessionTicket")
-        return "\"X-Authorization\", *mUserSessionTicket";
+        return "\"X-Authorization\", mUserSessionTicket";
```

Comment:
Hi @pgilmorepf.

Apologies for the confusion, our latest version does indeed revert the session ticket back to a static and still allows the session ticket to be unset. Overall it's a cleaner solution and safer as the pointer version was causing a crash. 

It was previously converted to a pointer because Aws::String has memory manager problems with shutting down statics. However now that we're using AZStd::string, the original pointer change is  no longer needed.